src/driver/cockroachdb/CockroachDriver.ts

```diff
@@ -479,7 +479,10 @@ export class CockroachDriver implements Driver {
         const defaultValue = columnMetadata.default;
         const arrayCast = columnMetadata.isArray ? `::${columnMetadata.type}[]` : "";
 
-        if (typeof defaultValue === "number") {
+        if (columnMetadata.isArray && Array.isArray(defaultValue)) {
+            return `'{${defaultValue.map((val: string) => `${val}`).join(",")}}'`;
```

Comment:
looks like it force casts array values to string even if column isn't a string-typed. What it can lead to? Also, tests are missing for this feature :) Thanks